### PR TITLE
 redirect to error template if there is an error whlie rendering the view

### PR DESCRIPTION
**Pull request by lalop**

### CODE REVIEW

File: Slim/View.php [hunk 1]
```diff
@@ -209,8 +209,13 @@ public function render($template)
         $this->setTemplate($template);
         extract($this->data);
         ob_start();
-        require $this->templatePath;
-
-        return ob_get_clean();
+        ob_start();
```
Comment (nubs) [inline Slim/View.php]: It looks like you got an extra ob_start in here.

**Comment by lalop** [comment]
Oo 
effectively, thank you

**Comment by codeguy** [comment]
Can you clarify what issue you are trying to fix?

**Comment by lalop** [comment]
I will try.
Today if an exception is thrown in the view the output buffer isn't turned off so the beginning of the view is send then it's the exception.
With my fix only the exception is send.

Is it more clear ?

**Comment by lalop** [comment]
If you want i can add some test on this feature ?

**Comment by lalop** [comment]
Do you need anything more on this point ? Information, tests ... ?

**Comment by lalop** [comment]
Hello,
what happens for this issue ?

**Comment by silentworks** [comment]
This seem to have been fixed now. Closing off.

**Pull request closed without merging**